website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -46,33 +57,8 @@ By default this will be the namespace that the operator is running in. To watch
 mgr, err := ctrl.NewManager(cfg, manager.Options{Namespace: ""})
 ```
 
-It is also possible to use the [MultiNamespacedCacheBuilder][multi-namespaced-cache-builder] to watch a specific set of namespaces:
-```Go
-var namespaces []string // List of Namespaces
-// Create a new Cmd to provide shared dependencies and start components
-mgr, err := ctrl.NewManager(cfg, manager.Options{
-   NewCache: cache.MultiNamespacedCacheBuilder(namespaces),
-})
-```
-
-#### Operator scope
-
 Read the [operator scope][operator_scope] documentation on how to run your operator as namespace-scoped vs cluster-scoped.
 
-### Multi-Group APIs
-
-Before creating an API and controller, consider if your operator requires multiple API [groups][api-groups]. Then to change the layout of your project to support multi-group run the command `operator-sdk edit --multigroup=true`. It will update the `PROJECT` file which should look like the following:
```

Comment:
Again, this is the more in depth tutorial, I think it would be better if we left this in?

Reply:
the tutorial is consumed for new users usually. Note that it does not use multigroup at all since it would be a more advanced step and only required in specific cases.  So, this explanation shows very confuse for who is trying to understand what is an operator. However, see that the info still available in the note:

<img width="901" alt="Screen Shot 2020-12-11 at 01 50 59" src="https://user-images.githubusercontent.com/7708031/101851388-5dfb3280-3b53-11eb-85e4-ae1697a5d3b1.png">